Berksfile

```diff
@@ -1,8 +1,12 @@
-source "https://api.berkshelf.com"
-cookbook "yum"
-cookbook "yum-epel"
-cookbook "user"
-cookbook "partial_search"
-cookbook "sudo"
-cookbook "nrpe"
+source 'https://api.berkshelf.com'
+cookbook 'yum'
+cookbook 'yum-epel'
+cookbook 'user'
+cookbook 'partial_search'
+cookbook 'sudo'
+cookbook 'nrpe'
```

Comment:
I thought we were removing all cookbooks that searchable in the supermarket from the Berksfile?